components/date-time/date.js

```diff
@@ -4,13 +4,21 @@
 import ReactDatePicker from 'react-datepicker';
 import moment from 'moment';
 
-export function DatePicker( { currentDate, onChange, ...args } ) {
+/**
+ * Module Constants
+ */
+const TIMEZONELESS_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
```

Comment:
Why not just `TIME_FORMAT`?

Reply:
The input and the output format should be the same, that's the format we have in the post object. We could use only a time format  for time and date format for date but this means adding some "unnecessary" date manipulations in the parent component `date-time` (extract time and date on input, and recompute the whole date on change). I think it's just for convenience.